EuiHeader Padding & EuiDescribedFormGroup Shrinking Bugs (#3369)
elastic/eui: CHANGELOG.md

- Fixed `EuiPopover` positioning from being overridden by `style` prop ([#3329](https://github.com/elastic/eui/pull/3329))
- Fixed `EuiCodeBlock` not copying updated content ([#3351](https://github.com/elastic/eui/pull/3351))
- Fixed alignment of popover of end date of `EuiDatePickerRange` ([#3359](https://github.com/elastic/eui/pull/3359))

> - Removed additional padding applied to body to account for `EuiHeader` when fixed ([#3369](https://github.com/elastic/eui/pull/3369))

Reviewer: Can you move this one out of bug fixes as it's not an actual bug? Also text makes me think it's **all** of the padding, not just the extra 4px.

```suggestion
- Removed additional padding applied to `$euiHeaderHeightCompensation` when `EuiHeader` is fixed ([#3369](https://github.com/elastic/eui/pull/3369))
```
